app/src/androidTest/java/org/mozilla/fenix/ui/BookmarksTest.kt

```diff
@@ -204,6 +256,25 @@ class BookmarksTest {
         }.openThreeDotMenu(defaultWebPage.url) {
         }.clickDelete {
             verifyDeleteSnackBarText()
+            verifyUndoDeleteSnackBarButton()
+        }
+    }
+
+    @Test
+    fun undoDeleteBookmarkTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        browserScreen {
+            createBookmark(defaultWebPage.url)
+        }.openThreeDotMenu {
+        }.openLibrary {
+        }.openBookmarks {
```

Comment:
Is an idling resource needed here too, because we're waiting for the populated list (even thought it's one bookmark)?

Reply:
I'll add it. Thanks.